Plugins/Flow.Launcher.Plugin.Sys/Main.cs

```diff
@@ -77,9 +82,50 @@ public List<Result> Query(Query query)
             return results;
         }
 
+        private string GetDynamicTitle(Query query, Result result)
+        {
+            var pair = KeywordTitleMappings
```

Comment:
why didn't use the dictionary accessor directly?

Reply:
Correct me if I'm wrong, but if you use the dictionary accessor directly, you can only perform a lookup based on the key, and you can't include the additional condition of inequality between key and value.
But acctually it might be better to use the accessor directly and make a new condition if key == value, return key and skip everything else.

Reply:
Now the value is the "key" to lookup, so it should never be equal to the key (the english version) right? or do I miss anything?

Reply:
Correct, the `value` (translation key) should never be equal to the `key` (english version), except if the translation is missing or the language is english. In this case theres no need to compare both matching scores, because the title is set to the english version anyways.